vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -972,9 +972,8 @@ class VectorPreferences @Inject constructor(private val context: Context) {
     }
 
     fun prefSpacesShowAllRoomInHome(): Boolean {
-        return defaultPrefs.getBoolean(SETTINGS_PREF_SPACE_SHOW_ALL_ROOM_IN_HOME,
-                // migration of old property
-                !labsSpacesOnlyOrphansInHome())
+        // Tchap: Show all in home until spaces are hidden
```

Comment:
maybe: Show all in the room. The spaces are hidden.

Reply:
I updated the doc which was indeed not exact, I think you meant `Show all in the home` instead